xds/internal/balancer/clusterresolver/config.go

```diff
@@ -168,18 +168,13 @@ type LBConfig struct {
 	XDSLBPolicy *internalserviceconfig.BalancerConfig `json:"xdsLbPolicy,omitempty"`
 }
 
-const (
-	rrName = roundrobin.Name
-	rhName = ringhash.Name
-)
-
 func parseConfig(c json.RawMessage) (*LBConfig, error) {
```

Comment:
Nothing calls this function anymore except the exported ParseConfig function (the ParseConfig test used to, but you changed it to call exported). Why do we even need this separate internal parseConfig method? Why not move this logic to exported function?

Reply:
Done.